config/datapass_webhooks_particulier.yml

```diff
@@ -36,17 +40,93 @@ test: *development
 production: &production
   created: &created
     emails:
+      - template: 'embarquement_brouillon_en_attente'
+        when: 'in 14 days'
+
   updated: &updated {}
   team_member_updated: &team_member_updated {}
   send_application: &send_application
     emails:
+      - template: 'demande_recue'
+
+      - template: 'reassurance_demande_recue'
+        when: 'in 14 days'
+
   notify: &notify {}
   review_application: &review_application
     emails:
+      - template: 'embarquement_modifications_demandees'
+
+      - template: 'embarquement_relance_modifications_demandees'
+        when: 'in 14 days'
+
   refuse_application: &refuse_application
     emails:
+      - template: 'embarquement_demande_refusee'
+
   validate_application: &validate_application
     emails:
+      - template: 'enquete_satisfaction'
```

Comment:
Globalement ici faut clean tous les templates pour retirer toute trace du contact métier.

Reply:
Et par ailleurs, à la lumière de mes commentaires plus bas on a un trou dans la raquette: on ne teste pas que ça fonctionne partout.. imo faudrait itérer sur spec/acceptances/datapass_webhooks_config_format_spec.rb pour vérifier que les méthodes existent sur chaque mailer.